packages/app-shell/src/components/AuthenticatedAppShellSideBarItem.tsx

```diff
@@ -0,0 +1,83 @@
+import { ComponentType } from 'react';
+import { Box, Flex } from '@ag.ds-next/react/box';
+import { boxPalette, tokens, useLinkComponent } from '@ag.ds-next/react/core';
+import { IconProps } from '@ag.ds-next/react/icon';
+import { NotificationBadge } from '@ag.ds-next/react/badge';
+
+export type AuthenticatedAppShellSideBarItemType = {
+	href: string;
+	label: string;
+	icon: ComponentType<IconProps>;
+	badgeCount?: number;
```

Comment:
Will we ever want to support anything more generic here than a badge? Could replace this with a ReactNode and open it up a lot for ourselves. Not sure if/how the designs may drift over time

```suggestion
	extraContent?: React.ReactNode;
```